Projects/Addons/Sources/Navigation/ChangeAddonNavigation.swift

```diff
@@ -29,7 +29,7 @@ class ChangeAddonNavigationViewModel: ObservableObject {
     @Published var isChangeCoverageDaysPresented: AddonOffer?
     @Published var isConfirmAddonPresented = false
     @Published var isAddonProcessingPresented = false
-    @Published var changeAddonVm: ChangeAddonViewModel
+    @Published var changeAddonVm: ChangeAddonViewModel?
```

Comment:
Added it as an optional since we can have changeAddonNavigation with list of contracts so we don't want to init it with empty contractId